src/features/virtualDocumentTracker.ts

```diff
@@ -3,27 +3,39 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import {workspace, TextDocument, Uri} from 'vscode';
-import {OmniSharpServer} from '../omnisharp/server';
+import { workspace, TextDocument, Uri } from 'vscode';
+import { OmniSharpServer } from '../omnisharp/server';
 import * as serverUtils from '../omnisharp/utils';
 import { FileChangeType } from '../omnisharp/protocol';
 import { IDisposable } from '../Disposable';
 import CompositeDisposable from '../CompositeDisposable';
 import { EventStream } from '../EventStream';
 import { DocumentSynchronizationFailure } from '../omnisharp/loggingEvents';
 
-function trackCurrentVirtualDocuments(server: OmniSharpServer, eventStream: EventStream) {
-    let registration = server.onProjectAdded(async () => {
```

Comment:
This isn't needed any more but because I removed it that also means we fire and forget the initial synchronization requests (aka we don't `await trackCurrentVirutalDocuments`). @rchande what's your opinions here?